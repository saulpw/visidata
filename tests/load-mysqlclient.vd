sheet	col	row	longname	input	keystrokes	comment
global		filetype	set-option	mysqlclient		
			open-file	sample_data/mysqlclient.tsv	o	
mysqlclient	LF		addcol-expr	ord(LF)	=	create new column from Python expression, with column names as variables
mysqlclient	BS		addcol-expr	ord(BS)	=	create new column from Python expression, with column names as variables
mysqlclient	TAB		addcol-expr	ord(TAB)	=	create new column from Python expression, with column names as variables
mysqlclient	CR		addcol-expr	ord(CR)	=	create new column from Python expression, with column names as variables
