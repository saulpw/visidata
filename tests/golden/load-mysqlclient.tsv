LF	ord(LF)	BS	ord(BS)	TAB	ord(TAB)	CR	ord(CR)

	10	\	92			9		13
1	49	2	50	3	51	4	52
